### ISSUE TITLE
[Feature request] Ask for download directory

### ISSUE THREAD

**Issue by jpbaril** [Lemutar/Synoloader#25]
I always place downloads in different directories by media types, e.g. music, movies, tv, etc.
I don't know if it's possible, but it would be useful if Synoloader could ask me in which subdirectory of main download directory I want to place the download. Or maybe it could open the Download Station webui starting torrent task dialog window. Thanks

**Comment by acorderob** [comment]
I agree. This would be nice to have, specially since the API allows it.

**Comment by noplanman** [comment]
Will check it out :blush:

**Comment by technodrome** [comment]
Any news on this? It really is a basic requirement.
